src/ribbon/page.cpp

```diff
@@ -1076,17 +1078,20 @@ bool wxRibbonPage::CollapsePanels(wxOrientation direction, int minimum_amount)
                     // come to this panel again anyway)
                     amount = 32;
                 }
-                if(direction & wxHORIZONTAL)
+                if (largest_panel_size)
```

Comment:
This change (and similar one below) doesn't seem to fall under cleanup/modernization, so if they're really needed, could you please resubmit them as a separate PR or at least a separate commit?

FWIW I think we should test for it above, instead of testing for `largest_panel`, if we really need to check it (the code is not very clear, so I'm not sure).

TIA!